RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Release Process
+jiva-operator follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects).
+
+## Release Candidate Verification Checklist
+
+Every release has release candidate builds that are created starting from the third week into the release. These release candidate builds help to freeze the scope and maintain the quality of the release. The release candidate builds will go through:
+- Platform Verification
+- Regression and Feature Verification Automated tests.
+- Exploratory testing by QA engineers
+- Strict security scanners on the container images
+- Upgrade from previous releases
+- Beta testing by users on issues that they are interested in.
+- Dogfooding on workload and MayaData OpenEBS Director staging cluster.
```

Comment:
This has been changed to mention about OpenEBS E2e cluster and not use director.

Reply:
done